### PR TITLE
Unconditionally enable local echo for emergency shell

### PR DESCRIPTION
**Pull request by nmeum**
Fixes https://github.com/anatol/booster/issues/144

### CODE REVIEW

File: init/main.go [hunk 1]
```diff
@@ -787,7 +787,30 @@ func readStartTime() {
...
+	if err := unix.IoctlSetTermios(fd, unix.TCSETS, &newState); err != nil {
+		return err
+	}
+
+	return nil
+}
+
 func emergencyShell() {
+	// Force local echo (might have been disabled by readPassword).
+	if err := enableLocalEcho(); err != nil {
```
Comment (anatol) [inline init/main.go:810]: let's move the `enableLocalEcho` call under `if _, err := os.Stat("/usr/bin/busybox"); !os.IsNotExist(err) {` so the echo will be disable only if it is needed.

**Review by anatol — COMMENTED** [review]
the rest looks good to me

File: init/main.go [hunk 2]
```diff
@@ -787,7 +787,30 @@ func readStartTime() {
...
+	fd := int(stdin.Fd())
+
+	termios, err := unix.IoctlGetTermios(fd, unix.TCGETS)
+	if err != nil {
+		return err
+	}
+
+	newState := *termios
+	newState.Lflag |= unix.ECHO
+	if err := unix.IoctlSetTermios(fd, unix.TCSETS, &newState); err != nil {
```
[the remark this answers is not in the document]
Comment (nmeum) [reply init/main.go]: Right, I just wasn't sure if this style is preferred. Adjusted accordingly, thank you!

File: init/main.go (hunk 1 above)
Comment (nmeum) [reply init/main.go]: Good catch, fixed.

**Comment by anatol** [comment]
Thank you!

**Pull request merged**